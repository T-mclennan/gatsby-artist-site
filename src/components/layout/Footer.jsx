import React from "react"
// import socialLinks from "../constants/social_links"
import './Footer.css'


const Footer = () => {
  return (<footer className="footer">
      <div>
        <h5>copyright&copy;{new Date().getFullYear()}
          <span>Graff and Stuff</span> all rights reserved
        </h5>
      </div>
    </footer>)
}

export default Footer