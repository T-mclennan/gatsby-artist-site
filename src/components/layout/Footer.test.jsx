import React from "react";
import renderer from "react-test-renderer";
import Footer from "./Footer";

describe("Footer", () => {
  it("Has properly rendered", () => {
    const tree = renderer
      .create(<Footer />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})