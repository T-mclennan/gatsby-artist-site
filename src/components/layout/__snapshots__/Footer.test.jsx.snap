// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Has properly rendered 1`] = `
<footer
  className="footer"
>
  <div>
    <h5>
      copyrightÂ©
      2022
      <span>
        Graff and Stuff
      </span>
       all rights reserved
    </h5>
  </div>
</footer>
`;
